manifest_template.yml

```diff
@@ -27,9 +27,19 @@ APIGEE_ENVIRONMENTS:
   quota: '7800'
 
 INTERNAL_DEV_VARIANTS:
-- alpha
-- rc
-- fix
+- name: alpha
+- name: rc
+- name: fix
+- name: ft01
+  service: fti
```

Comment:
I assume FT is new environment and never had existing product in it. if existing products are there, need to migrate the apps.

Reply:
Hi Raja, we have a new "FT" app that supports access to the FT environments - this also sets out the correct ASID etc.

Reply:
.. The FT environments have to be present in the manifest, but in fact - we only "use" them in the PR pipeline definition - so its for development only - and the PR pipeline creates unique products for the PR.